### PR TITLE
vim-patch: update runtime

### LINKED ISSUE #10572: Error detected while processing function netrw#Explore[220]
- `nvim --version`: NVIM v0.4.0-dev
<details>
<summary>Output of nvim --version </summary>

```
NVIM v0.4.0-dev
Build type: RelWithDebInfo
Lua 5.1
Compilation: /usr/bin/x86_64-linux-gnu-gcc -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -g -DMIN_LOG_LEVEL=3 -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wvla -fstack-protector-strong -fdiagnostics-color=auto -Wno-array-bounds -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -I/build/neovim-HiXy_F/neovim-0.4.0+ubuntu4+git201907212017-9d0f822-6e51973/build/config -I/build/neovim-HiXy_F/neovim-0.4.0+ubuntu4+git201907212017-9d0f822-6e51973/src -I/build/neovim-HiXy_F/neovim-0.4.0+ubuntu4+git201907212017-9d0f822-6e51973/.deps/usr/include -I/usr/include -I/build/neovim-HiXy_F/neovim-0.4.0+ubuntu4+git201907212017-9d0f822-6e51973/build/src/nvim/auto -I/build/neovim-HiXy_F/neovim-0.4.0+ubuntu4+git201907212017-9d0f822-6e51973/build/include
Compiled by buildd@lgw01-amd64-025
Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

</details>

- `vim -u DEFAULTS` (version: ) behaves differently? Works with vanilla vim, not with minimal init.vim on nvim. 
- Operating system/version: Ubuntu 16.04
- Terminal name/version: Remote SSH section
- `$TERM`: xterm-256color
 
### Steps to reproduce using `nvim -u NORC`
```
nvim -u NORC
:Explore 
```

### Actual behaviour

```
Error detected while processing function netrw#Explore[220]..netrw#LocalBrowseCheck:
line   23:
E171: Missing :endif
Error detected while processing function netrw#Explore:
line  220:
E171: Missing :endif
```

### Expected behaviour

Netrw doesn't yield error messages

### PR DESCRIPTION
**Pull request by justinmk**
fix #10572

### CODE REVIEW

File: runtime/doc/pi_netrw.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
-*pi_netrw.txt*  Nvim
+*pi_netrw.txt*  For Vim version 8.1.  Last change: 2019 Jul 17
```
Comment (zeertzjq) [inline runtime/doc/pi_netrw.txt:1]: Why is this line changed in this PR?

Comment (justinmk) [reply runtime/doc/pi_netrw.txt:1]: Why not? Likely I was intending to bring netrw closer to upstream.

Comment (zeertzjq) [reply runtime/doc/pi_netrw.txt:1]: Shouldn't an exception be added to `vim-patch.sh` to keep this line in diff then?

Comment (justinmk) [reply runtime/doc/pi_netrw.txt:1]: The less we do in service of netrw, the better. I wouldn't favor a netrw-specific code in vim-patch.sh unless it results in a net reduction of netrw-related work :)

Comment (zeertzjq) [reply runtime/doc/pi_netrw.txt:1]: If no exception is added to `vim-patch.sh`, this line almost never gets updated, and is constantly out-of-date. Currently in Nvim it's:
```
*pi_netrw.txt*  For Vim version 8.2.  Last change: 2020 Aug 15
```
While in Vim it's:
```
*pi_netrw.txt*  For Vim version 9.1.  Last change: 2024 Nov 02
```
Even though the remaining parts of `pi_netrw.txt` is almost the same.

So there are two options:
1. Remove the date
2. Add an exception to `vim-patch.sh` so that the date update in `pi_netrw.txt` is no longer ignored.

Comment (justinmk) [reply runtime/doc/pi_netrw.txt:1]: I'd lean towards removing the date.
